src/qml/QFieldCloudScreen.qml

```diff
@@ -594,12 +595,12 @@ Page {
     }
   }
 
-  function refreshProjectsList(shouldRefreshPublic) {
+  function refreshProjectsList(shouldRefreshPublic, resetOffset) {
```

Comment:
We're getting rid of the offset reset parameter, you'll just need to pass on the one public vs private boolean like before).